src/main/resources/flows-config.yaml

```diff
@@ -299,11 +299,15 @@ flow:
   confirmation:
     beforeDisplayAction: FormatSubmittedAtDate
     nextScreens:
-      - name: confirmation
+      - name: finalPage
+  finalPage:
+    nextScreens:
+      - name: finalPage
```

Comment:
This looks like an endless loop...?

Reply:
Yeah, it should be okay though, as the finalPage doesn't have any way to "submit". It will redirect to the index page after 5 seconds.